extensions/amp-story/0.1/amp-story.css

```diff
@@ -14,5 +14,459 @@
  * limitations under the License.
  */
 
+/** Common */
+amp-story, amp-story-page, amp-story-grid-layer {
+  contain: strict !important;
+  overflow: hidden !important;
+}
+
+/** Story level */
 amp-story {
+  height: 100% !important;
+  position: relative !important;
+  text-rendering: geometricPrecision !important;
+  user-select: none !important;
+  width: 100% !important;
+}
+
+html.i-amphtml-story-standalone,
+html[amp-story],
+html[📖],
+html.i-amphtml-story-standalone body,
+html[amp-story] body,
+html[📖] body {
+  height: 100% !important;
+  margin: 0 !important;
+  padding: 0 !important;
+  width: 100% !important;
+}
+
+html.i-amphtml-story-standalone body,
+html[amp-story] body,
+html[📖] body  {
+  display: grid !important;
```

Comment:
Does the body need be grid as well?

Reply:
This is `display: grid` for a different reason; this is our current "desktop UI," which is just a centered mobile-phone-sized screen.  This will be removed soon, when we have a true desktop UI (that is being developed now).